pkgs/development/python-modules/psycopg/default.nix

```diff
@@ -216,6 +216,8 @@ buildPythonPackage rec {
     # https://github.com/psycopg/psycopg/pull/915
     "tests/test_notify.py"
     "tests/test_notify_async.py"
+
+    "tests/test_concurrency.py"
```

Comment:
> * Make sure the tests are enabled using for example [`pytestCheckHook`](#using-pytestcheckhook) and, in the case of
  libraries, are passing for all interpreters. If certain tests fail they can be
  disabled individually. Try to avoid disabling the tests altogether. In any
  case, when you disable tests, leave a comment explaining not only _what_ the failure
  is but _why_ the test failure can be ignored for safe distribution with nixpkgs.

quoted from the Python section of the manual.